.github/workflows/pre-commit-status.yml

```diff
@@ -131,3 +131,34 @@ jobs:
           gh pr edit ${{ steps.pr-info.outputs.pr_number }} --repo ${{ github.repository }} --remove-label 'Status: Pre-commit fixes required ⚠️'
         env:
           GH_TOKEN: ${{ github.token }}
+
+      - name: Comment on PR about pre-commit failures
+        if: |
+          steps.pr-info.outputs.artifacts_found == 'true' &&
+          steps.pr-info.outputs.pre_commit_outcome == 'failure' &&
+          steps.pr-info.outputs.pending_commit == '0'
+        uses: thollander/actions-comment-pull-request@24bffb9b452ba05a4f3f77933840a6a841d1b32b # v3.0.1
+        with:
+          pr-number: ${{ steps.pr-info.outputs.pr_number }}
+          comment-tag: pre-commit-failure
+          message: |
+            ## ⚠️ Pre-commit Hooks Failed
+
+            Some pre-commit hooks failed and require manual fixes.
+
+            **What to do:**
+            1. 🔧 Fix the issues locally in your code
+            2. 💾 Commit and push your changes
+            3. 🔄 The hooks will run again automatically
+
+            [🔗 View detailed error log](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.event.workflow_run.id }})
+
+      - name: Remove pre-commit failure comment if resolved
```

Comment:
I don't think it's necessary to remove the message. It's a good history and link to what went wrong